cabal-install/Distribution/Client/Setup.hs

```diff
@@ -196,6 +200,11 @@ globalCommand commands = CommandUI {
           , "outdated"
           , "haddock"
           , "hscolour"
+          , "copy"
```

Comment:
I don't really know what that is supposed to be.

Reply:
Deleted, these somehow slipped into to rebase, they were not present in the initial pull request.